src/models/thelio-mira-r3/README.md

```diff
@@ -0,0 +1,97 @@
+# Thelio Mira (thelio-mira-r3)
+
+- [External Overview](./external-overview.md)
+- [Internal Overview](./internal-overview.md)
+- [Parts & Repairs](./repairs.md)
+
+![Thelio](./img/thelio-mira-r3.webp)
+
+The System76 Thelio Mira is a desktop with the following specifications:
+
+- Processor options
+    - Supports AMD Ryzen 7000 series (Zen 4) CPUs
+        - [Ryzen 5 7600X](https://www.amd.com/en/products/cpu/amd-ryzen-5-7600x#product-specs)
+        - [Ryzen 7 7700X](https://www.amd.com/en/products/cpu/amd-ryzen-7-7700x#product-specs)
+        - [Ryzen 9 7900X](https://www.amd.com/en/products/cpu/amd-ryzen-9-7900x#product-specs)
+        - [Ryzen 9 7950X](https://www.amd.com/en/products/cpu/amd-ryzen-9-7950x#product-specs)
+- Motherboard
+    - [Gigabyte X670 Aorus Elite AX](https://www.gigabyte.com/Motherboard/X670-AORUS-ELITE-AX-rev-11/sp) running System76 Firmware (non-open)
+    - AMD X670 chipset
+- Daughterboard
+    - [Thelio Io](https://github.com/system76/thelio-io) board running [open-source firmware](https://github.com/system76/thelio-io-firmware)
+- Graphics options
+    - Two GPUs of up to two-slot height
```

Comment:
I tried a different way of phrasing this bullet point in https://github.com/system76/tech-docs/pull/158/commits/0154dce4fc8bc44e0e1ee6bbee0784634293b612, let me know what you think. Not sure how into the weeds we should get since people could theoretically do a lot of crazy configurations on most of our systems if they're willing to run single-slot cards at tiny bandwidth. (There's also the question of what separates this section from the "Expansion" section lower down, and if we want to put stipulations about which slots become unavailable when installing what in other slots... but the photos on the other pages answer a lot of these questions easier.)

This was a good thing to call out since 4090s can extend above 2 slots.

Reply:
That looks good to me!